_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5081,10 +5081,10 @@ mfxU64 make_back_color_yuv(mfxU16 bit_depth, mfxU16 Y, mfxU16 U, mfxU16 V)
     mfxU64 const max_val = 256 << shift;
 
     return
-        ((mfxU64)                                 (max_val - 1) << 48) |
-        ((mfxU64)VPP_RANGE_CLIP(Y, (16 << shift), (235 << shift))     << 32) |
-        ((mfxU64)VPP_RANGE_CLIP(U, (16 << shift), (240 << shift))     << 16) |
-        ((mfxU64)VPP_RANGE_CLIP(V, (16 << shift), (240 << shift))     <<  0)
+        ((mfxU64)             (max_val - 1)  << 48)  |
+        ((mfxU64)VPP_RANGE_CLIP(Y, 16, 235)  << 32)  |
+        ((mfxU64)VPP_RANGE_CLIP(U, 16, 240)  << 16)  |
+        ((mfxU64)VPP_RANGE_CLIP(V, 16, 240)  <<  0)
```

Comment:
Here, for example 10 bit value is taken and clipped as it were 8 bit. It will be dark-green world!